app/assets/javascripts/fixthemap.js

```diff
@@ -1,8 +1,8 @@
 $(function () {
   const params = OSM.params();
 
-  let url = "/note/new";
   if (!params.zoom) params.zoom = 17;
-  if (params.lat && params.lon) url += OSM.formatHash(params);
-  $(".icon.note").attr("href", url);
+  if (params.lat && params.lon) {
+    $("a[href*='note']")[0].href += OSM.formatHash(params);
```

Comment:
This is rather nasty as it means this code is having to make assumptions about the URL that has been filled in by the server, plus we're hoping that no other links contain `note` anywhere... Can we not mark the element with a class that can be used to pick it out precisely?